tools/CMakeLists.txt

```
@@ -44,6 +46,7 @@ set(CMAKE_CXX_STANDARD 14)
 add_executable(sdbus-c++-xml2cpp ${SDBUSCPP_XML2CPP_SRCS})
 target_link_libraries (sdbus-c++-xml2cpp ${EXPAT_LIBRARIES})
 target_include_directories(sdbus-c++-xml2cpp PRIVATE ${EXPAT_INCLUDE_DIRS})
+target_compile_definitions(sdbus-c++-xml2cpp PRIVATE SDBUS_XML2CPP_VERSION="${sdbus-c++-tools_VERSION}")
```

Can't we use ${CMAKE_PROJECT_VERSION} here directly?

```suggestion
target_compile_definitions(sdbus-c++-xml2cpp PRIVATE SDBUS_XML2CPP_VERSION="${CMAKE_PROJECT_VERSION}")
```

Then we can drop unnecessary `sdbus-c++-tools_VERSION` variable.

---

Reading on `CMAKE_PROJECT_VERSION` in [the cmake documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_PROJECT_VERSION.html) I found the following line:
```
In the event that the top level CMakeLists.txt contains multiple [project()](https://cmake.org/cmake/help/latest/command/project.html#command:project) calls, the most recently called one from that top level CMakeLists.txt will determine the value that CMAKE_PROJECT_VERSION contains.
```

So I decided to first grab the variable right after the project definition. If you consider this unnecessary then I can update the code accordingly.

---

Yes, this is unnecessary, since there are no other `project()` calls in between the latest `project()` call and `target_compile_definitions()`. Please update as suggested...

---

Done

---

The is twice the dollar sign -- this is not intended, right?